x-pack/plugins/cloud_security_posture/public/pages/benchmarks/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// TODO: Use interface from BE https://github.com/elastic/security-team/issues/2942
```

Comment:
nit: We generally try to avoid referring to internal repos in our code or in public-facing issues.